modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/upgrade-catch-all-check/LPD_8024.testjava

```diff
@@ -0,0 +1,17 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+
+/**
+ * @author Renan Vasconcelos
+ */
+public class LPD_8024 {
+
+	public method(String key, InfoLocalizedValue<String> labelInfoLocalizedValue) {
```

Comment:
@RenanUmV SF, please break the line as the sf would do it for method declarations